Library/Formula/shibboleth-sp.rb

```diff
@@ -0,0 +1,85 @@
+class ShibbolethSp < Formula
+  desc "Shibboleth 2 Service Provider daemon"
+  homepage "https://wiki.shibboleth.net/confluence/display/SHIB2"
+  url "http://shibboleth.net/downloads/service-provider/latest/shibboleth-sp-2.5.5.tar.gz"
+  sha256 "30da36e0bba2ce4606a9effc37c05cd110dafdd6d3141468c4aa0f57ce4d96ce"
+
+  option "with-apache-22" # selecting this option compiles mod_shib_22.so
```

Comment:
You need to actually add the comment to the option, not beside it. It's user-facing, not specifically contributor-facing.